src/components/css/Table.css

```diff
@@ -10,4 +10,20 @@ thead th:hover {
     cursor: pointer;
 }
 
+html, body {
```

Comment:
what is the use of setting the css for html and body?

Reply:
Doesn't seem to have any usage... I have deleted them.